test/integration/src/test/java/io/pravega/test/integration/endtoendtest/ScopeTest.java

```diff
@@ -290,4 +290,79 @@ public void testForceDeleteScope() throws Exception {
 
         assertTrue(streamManager.deleteScope(scope, true));
     }
+
+    @Test
+    public void testDeleteScopeRecursive() throws Exception {
```

Comment:
Please add more Integration tests to cover all possible usage scenarios.

Reply:
tests have been added

Reply:
I dont; see any new tests.